workers/go/worker/worker.go

```diff
@@ -53,12 +52,17 @@ func (a *App) Run(cmd *cobra.Command, args []string) {
 
 func runWorkers(client client.Client, taskQueues []string) error {
 	errCh := make(chan error, len(taskQueues))
+	activityS := activities.Activities{
```

Comment:
```suggestion
	activityInstance := activities.Activities{
```

 (nitpick, can ignore)